x-pack/plugins/triggers_actions_ui/public/application/components/update_api_key_modal_confirmation.tsx

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
**question:** should we create a `update_api_key_modal_confirmation.tsx` file with some tests?

Reply:
It's all functionality is tested with `rule_list.test.tsx` didn't want to test the same thing again.
Actually i was planning to discuss about switching to react-testing-library gradually in all the tests. With RTL, testing the parent component with all the functionality is enough.

Reply:
Ah, I see. Is there an advantage to testing that functionality there instead of a sibling unit test file? That test file may need to be broken down in the future to keep the line count low.

Reply:
RTL It's about focusing on the outputs and behaviours, by testing the main component we can test all the expected behaviours and their outputs. In this case i've tested all the possible actions in `rule_list` and didn't repeat them in rule_details even though it uses the same component. 

I don't care what the confirm modal's prop name is (e.g. apiUpdateApiKeyCall) as long as it works well with the parent component and i don't need to update any test when i change that prop name... My subcutaneous test gives me the confidence that nothing is broken :)

But, since we added conditional translations and support for bulk update, and didn't test them in the parent component (because it doesn't support bulk update yet), testing it separately might be better :)

Reply:
Not a problem, thanks for the explanation 🙏 I will resolve this thread.